src/Uno.UI/UI/Xaml/Media/BrushMappingMode.cs

```diff
@@ -0,0 +1,10 @@
+namespace Windows.UI.Xaml.Media
+{
+	public enum BrushMappingMode 
+	{
+		[Uno.NotImplemented]
```

Comment:
I don't really understand how 1 out of 2 `enum` values can be implemented, given that you have a conditional on `== RelativeToBoundingBox` below.